csharp/src/AddressBook/Properties/AssemblyInfo.cs

```diff
@@ -25,5 +25,5 @@
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("2.4.1.555")]
 
-[assembly: AssemblyVersion("2.4.1.555")]
-[assembly: AssemblyFileVersion("2.4.1.555")]
\ No newline at end of file
+[assembly: AssemblyVersion("3.0.0.1")]

```

Comment:
We can use AssemblyInformationalVersion for the alpha part.
Actually, I think we should be using AssemblyVersion 3.0.0 throughout the 3.x line, and use AssemblyFileVersion for the more detailed version number. I'll see if I can dig out a Stack Overflow answer giving the rationale for that.